### DISCUSSION TITLE
v54

### DISCUSSION THREAD

**Discussion by andyholmes** [andyholmes/gnome-shell-extension-gsconnect#1488] [Announcements]
**v54** is now available for download.

**Notable changes & fixes**
* New icon (#1477 by @daudix-UFO) :trophy:
* Port to GNOME 43 (#1475 by @MacTavishAO, #1441 by @jbicha) :trophy:
* Add remote input touchpad (#1469 by @WSID) :trophy:

As always, thanks to contributors of all kinds to GSConnect, and the KDE Connect team!

List of changes since **v50** [here](https://github.com/GSConnect/gnome-shell-extension-gsconnect/compare/v50...v54).

<hr /><em>This discussion was created from the release <a href='https://github.com/GSConnect/gnome-shell-extension-gsconnect/releases/tag/v54'>v54</a>.</em>

**Comment by daniellandau** [comment]
Now published in extensions.gnome.org, thank you all!
